VideoLocker/res/values/dimens.xml

```diff
@@ -43,7 +43,9 @@
 
     <!-- Course Card -->
     <dimen name="course_card_height">190dp</dimen>
+    <dimen name="course_card_play_icon_diameter">60dp</dimen>
     <dimen name="course_detail_card_height">70dp</dimen>
+    <dimen name="course_detail_share_icon_diameter">36dp</dimen>
```

Comment:
I think replace "diameter" with "size" to keep the naming scheme consistent with other ones.